src/emulator/host/src/host.cpp

```diff
@@ -27,7 +27,7 @@
 #include <unordered_set>
 
 #ifdef NDEBUG // Leave it as non-constexpr on Debug so that we can enable/disable it at will via set_log_import_calls
-    constexpr
+constexpr
```

Comment:
Why was this changed? I have already run clang-format for this file. Make sure yours is using our `.clang-format`.

Reply:
i have just run format.bat like all time before so no idea

Reply:
i have revert it

Reply:
I re-run it and it did the same... Not sure what's up, re-run it and include all changes it does to your commit.

Reply:
oki, is done